client/flutter/lib/api/user_preferences.dart

```diff
@@ -44,6 +44,10 @@ class UserPreferences {
         .setBool(UserPreferenceKey.AnalyticsEnabled.toString(), value);
   }
 
+  Future<bool> getTermsOfServiceAccepted() async {
+    return getAnalyticsEnabled();
```

Comment:
Is this correct? A user can accept terms of service and then disable analytics later.

Reply:
You are right, it's not correct. Will fix.

Reply:
Ok, I've added a real "terms of service" preference to track the legal page. Please note that to properly test this you'll have to reinstall the app as this flag is set only once during onboarding.